docs/v0.4/getting-started/docker-for-mac.md

```diff
@@ -6,16 +6,17 @@ sidebar_label: Docker for Mac
 
 The following will help you get started running a riff function with Knative on Docker Community Edition for Mac.
 
-### TL;DR
+## TL;DR
```

Comment:
Can we drop the tl;dr now that the pagenav has similar content (with links)?

Reply:
👍 yes - was thinking the same.